discoverable-client/src/test/java/com/ca/mfaas/client/api/PetControllerPutTest.java

```diff
@@ -27,11 +27,7 @@
 
 import static org.hamcrest.Matchers.hasItem;
 import static org.hamcrest.core.Is.is;
-import static org.mockito.Mockito.any;
-import static org.mockito.Mockito.never;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
The same... It's better not import every static method from `org.mockito.Mockito`, because there are too many, but we need only few of them.

Reply:
Fixed this one too